hera_sim/visibilities/prisim_wrapper.py

```diff
@@ -279,11 +303,14 @@ def _simulate(self):
                 roi_ind_snap = np.array([])
                 roi_pbeam_snap = np.array([])
 
+            # UVData has a bandpass_array, but AnalyticBeams do not
+            bandpass = getattr(self.beams[0], "bandpass_array", np.ones_like(self.uvdata.freq_array[0]))
+
             self.interferometer.observe(
                 timeobj=self._astropy_times[j],
                 Tsysinfo={"Tnet": 0},
-                bandpass=self.beams[0].bandpass_array,
-                pointing_center=np.array([90.0, 270.0]),
+                bandpass=bandpass,
+                pointing_center=np.array([90.0, 270.0]), ### ASK      
```

Comment:
what did you want to ask?

Reply:
There was an issue regarding the shape, but we pushed off discussing the `PRISim` wrapper implementation until later anyways.

Reply:
Yeah, cool. Actually, I think once we merge this, I'll remove the PRISIM wrapper from the branch into its own branch for later.